solutions/security/ai/attack-discovery.md

```diff
@@ -106,3 +99,67 @@ There are several ways you can incorporate discoveries into your {{elastic-sec}}
 :::{image} /solutions/images/security-add-discovery-to-assistant.gif
 :alt: Attack Discovery view in AI Assistant
 :::
+
+## Schedule discoveries
+
+```yaml {applies_to}
+stack: ga 9.1
+serverless: ga
+```
+
+You can define recurring schedules (for example, daily or weekly) to automatically generate attack discoveries without needing manual runs. For example, you can generate discoveries every 24 hours and send a Slack notification to your SecOps channel if discoveries are found. Notifications are sent using configured [connectors](/deploy-manage/manage-connectors.md), such as Slack or email, and you can customize the notification content to tailor alert context to your needs.
+
+Scheduled discoveries are shared by default with all users in a {{kib}} space. 
+
+:::{note}
+You can still generate discoveries manually at any time, regardless of an active schedule.
+:::
+
+To create a new schedule:
+
+1. Click the gear icon to open the settings menu, then select **Schedule**.
+2. Select **Create new schedule**.
+3. Enter a name for the new schedule.
+4. Select the LLM connector to use for generating discoveries, or add a new one.
+5. Use the KQL query bar, time filter, and alerts slider to customize the set of alerts that will be analyzed.
+6. Define the schedule's frequency (for example, every 24 hours).
+7. Optionally, select the [connectors](/deploy-manage/manage-connectors.md) to use for receiving notifications, and define their actions.
+8. Click **Create & enable schedule**.
+
+After creating new schedules, you can view their status, modify them or delete them from the **Schedule** tab in the settings menu.
+
+:::{tip}
+Scheduled discoveries are shown with a **Scheduled Attack discovery** icon ({icon}`calendar`). Click the icon to view the schedule that created it.
+:::
+
+## View saved discoveries
+
+```yaml {applies_to}
+stack: ga 9.1
+serverless: ga
+```
+
+Attack discoveries are automatically saved each time you generate them. Once saved, discoveries remain available for later review, reporting, and tracking over time. This allows you to revisit discoveries to monitor trends, maintain audit trails, and support investigations as your environment evolves.
```

Comment:
I found myself wondering where the shared discoveries appear. Perhaps it's obvious in the UI, so feel free to ignore this, but it might be worth adding a sentence describing where the discoveries are saved.